edivorce/apps/core/serializer.py

```diff
@@ -3,6 +3,7 @@
 from rest_framework.exceptions import ValidationError
 
 from .models import Document, UserResponse
+from .validators import *
```

Comment:
When you do `import *`, it can cause a bunch of side effects. Better to import each validator individually, or `from . import validators`